onnxmltools/convert/common/_topology.py

```diff
@@ -648,6 +649,19 @@ def convert_topology(topology, model_name, doc_string, targeted_onnx):
                 else:
                     other_outputs[variable.raw_name] = variable
 
+    # Check input and output naming convention
+    for name in topology.raw_model.input_names:
```

Comment:
Do every model (CoreML, keras, scikit-learn) have this property - input_names?

Reply:
`raw_model` is a wrapper class over the targeted source model. For Core ML model, its type is `CoremlModelContainer` defined [here](https://github.com/onnx/onnxmltools/blob/727cc8fadfbc17790978bb6c66db461ff62c764d/onnxmltools/convert/common/_container.py#L43). Similarly, we have `SklearnModelContainer` and `KerasModelContainer` for the other two frameworks. Please see [`RawModelContainer`](https://github.com/onnx/onnxmltools/blob/727cc8fadfbc17790978bb6c66db461ff62c764d/onnxmltools/convert/common/_container.py#L13) for their common behavior.